CHANGELOG.md

```diff
@@ -1,5 +1,13 @@
 # Revision history for plutarch
 
+# Unreleased
+
+- `TermCont`: Parametrize by result type; add `MonadFail` instance; etc.
```

Comment:
Can we have a bit more info on here? It's not actually explicitly parameterized by result type now.

Reply:
I think this wording is fine.